app-games/src/main/java/com/aptoide/android/aptoidegames/updates/UpdatesScreen.kt

```diff
@@ -83,3 +100,86 @@ fun NoUpdatesScreen() {
     )
   }
 }
+
+@Composable
+private fun AppsList(
+  appList: List<App>,
+  navigate: (String) -> Unit,
+) {
+  Column(
+    modifier = Modifier
+      .fillMaxWidth(),
+    horizontalAlignment = Alignment.CenterHorizontally
+  ) {
+    UpdateBox()
+    LazyColumn(
+      modifier = Modifier
+        .semantics { collectionInfo = CollectionInfo(appList.size, 1) }
```

Comment:
Even though there are no accessibility requirements, this semantics are wrong. The column and row info should be switched

Reply:
@Culurio this comment was wrong sorry, the change should be reverted 😅